frontend/tailwind.config.js

```diff
@@ -1,8 +1,16 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  darkMode: 'media',
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        dark: {
+          background: 'var(--color-dark-background)',
+          background_card_textfield: 'var(--color-dark-textfield-card)',
```

Comment:
While everything else looks good to me. The name of this variable is the only thing that bothers me.. a lot 😛 

You can probably translate `background_card_textfield` to something like this:

```js
extend: {
      colors: {
        dark: {
          background: {
		DEFAULT: 'var(--color-dark-background)',
		card: {
			textfield: 'var(--color-dark-textfield-card)'
		}
	},
```

This will allow you to use `dark:bg-dark-background-card-textfield`

But, if feel "background" here is redundant and I think is against Tailwind's naming conventions.
How about if we do `dark:bg-dark-textfield`? Like this:

```js
extend: {
      colors: {
        dark: {
		DEFAULT: 'var(--color-dark-background)',
		textfield: 'var(--color-dark-textfield-card)'
	},
```

You're the better judge here, I'll leave it upto you :)
### I'm not sure if we're using this name somewhere already. So it's best to do a little searching before the implementation.

Reply:
@chinmaykunkikar Hi,
I did some search and I found 
 extend: {
      colors: {
        dark: {
          background: {
            DEFAULT: 'var(--color-dark-background)',
            card: {
              textfield: 'var(--color-dark-textfield-card)',
            },
          },
        },
      },
    },
this way. I think this way will help to help other contributer in understanding.

and now I used it like " dark:bg-dark-background-card-textfield " and "dark:bg-dark-background "

And in above approach which  I use I  thought it may help other to understand.
can u review

Reply:
That looks good. You can go ahead with the suggestions now :)

Reply:
@chinmaykunkikar  I haven't updated PR as it is same

Reply:
Yeah, it wasn't needed. Thanks for the changes.

Reply:
@SankalpHaritash21, I actually agree with @chinmaykunkikar , background is redundant. Can we remove the background layer and use it like

```
extend: {
      colors: {
        dark: {
		DEFAULT: 'var(--color-dark-background)',
		textfield: 'var(--color-dark-textfield-card)'
	},
```
As suggested... This looks neat to me as we remove one more level of branching as background which is presently not required because `bg-dark-textfield` and `bg-dark` is more readable and better understandable than `dark:bg-dark-background-card-textfield` and `dark:bg-dark-background`

Kindly make the required changes :)

Reply:
sir can we keep "dark:bg-dark-background-textfield" background . Tryed but it creates a confusion. @krishnaacharyaa  its ur last decision.

Reply:
Please explain me why `dark:bg-dark-background-textfield` causes confusion and how is it better than @chinmaykunkikar's suggestion, will consider if convinced :) Because I feel the above is much clean and better to understand

Reply:
@krishnaacharyaa  I mean to say that is this background layer will help us to keep difference if we make any other changes in future then we can add a new different layer in easily in dark other wise we need to make changes in each file. 
As this background layer may seem to be irrelevant but it help to indicate background  color. 
Otherwise it create confusion as text-dark-textfield as we can use these color variable with both bg and text.

Reply:
@SankalpHaritash21, Somehow i'm not convinced, I understand your concern, we'll change it when required in the future because for immediate future I dont see any need for the same, kindly make the change

Reply:
@krishnaacharyaa I have updated the code plz check